### PR TITLE
Import statement is missing

### PR DESCRIPTION
**Pull request by navba-MSFT**
The below Import package statement code is missing in [our docs](https://learn.microsoft.com/en-us/python/api/overview/azure/digitaltwins-core-readme?view=azure-python). 

```
from azure.identity import DefaultAzureCredential
from azure.digitaltwins.core import DigitalTwinsClient
```

Also having the below samples link in the article would be useful:
https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/digitaltwins/azure-mgmt-digitaltwins/generated_samples

Fixes Azure/azure-sdk-for-python#30855

# Description

Please add an informative description that covers that changes made by the pull request and link all relevant issues.

If an SDK is being regenerated based on a new swagger spec, a link to the pull request containing these swagger spec changes has been included above.

# All SDK Contribution checklist:
- [x] **The pull request does not introduce [breaking changes]**
- [x] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md).**

## General Guidelines and Best Practices
- [ ] Title of the pull request is clear and informative.
- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](https://github.com/Azure/azure-sdk-for-python/blob/main/CONTRIBUTING.md##building-and-testing)
- [ ] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by navba-MSFT** [comment]
@johngallardo @Aashish93-stack @sjiherzig Could you please review this PR once you get a chance ? Thanks in advance.

File: sdk/digitaltwins/azure-digitaltwins-core/README.md [hunk 1]
```diff
@@ -28,6 +28,16 @@ authentication as demonstrated below.
 
 ### How to use
 
+#### Import the packages
```
Comment (sjiherzig) [inline sdk/digitaltwins/azure-digitaltwins-core/README.md:31]: I'd prefer including the imports in the sample code for authentication and creating the DT client further below.

File: sdk/digitaltwins/azure-digitaltwins-core/README.md [hunk 2]
```diff
@@ -65,7 +75,7 @@ You can learn more about Azure Digital Twins by visiting [Azure Digital Twins Do
 
 ## Examples
 
-You can explore the digital twins APIs (using the client library) using the samples project.
+You can explore the digital twins APIs (using the client library) using the samples project. More samples on DigitalTwins are available [here](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/digitaltwins/azure-mgmt-digitaltwins/generated_samples).
```
Comment (sjiherzig) [inline sdk/digitaltwins/azure-digitaltwins-core/README.md:78]: I'm surprised the generated samples aren't part of the samples project already. Can we not add them instead? Having samples in two places is a bit confusing.

**Review by sjiherzig — CHANGES_REQUESTED** [review]

File: sdk/digitaltwins/azure-digitaltwins-core/README.md (hunk 1 above)
Comment (navba-MSFT) [reply sdk/digitaltwins/azure-digitaltwins-core/README.md:31]: done.

File: sdk/digitaltwins/azure-digitaltwins-core/README.md (hunk 2 above)
Comment (navba-MSFT) [reply sdk/digitaltwins/azure-digitaltwins-core/README.md:71]: The currently existing docs, helps in getting started with key elements of Azure Digital Twins, such as creating Azure Digital Twins instances, models, twin graphs, etc. Also the samples provided here guides to familiarize yourself with the various APIs that help you program against Azure Digital Twins. So we dont want to include all of the generated 30+ samples from the github here.

Comment (msyyc) [reply sdk/digitaltwins/azure-digitaltwins-core/README.md:71]: It may be confusing to involve samples of mgmt SDK in data-plane SDK readme.md

Comment (navba-MSFT) [reply sdk/digitaltwins/azure-digitaltwins-core/README.md]: @msyyc That makes sense. So I have reverted it.

**Review by msyyc — APPROVED** [review]

**Review by sjiherzig — APPROVED** [review]

**Pull request merged**